app/models/wpcc/wpcc_config.rb

```diff
@@ -29,7 +29,7 @@ def min_contribution_percentages_by_period
     private
 
     def current_period
-      'april_2018_march_2019'
+      'current'
```

Comment:
have you considered to convert this into a `CURRENT_PERIOD` constant? A literal string with no logic... seems the perfect fit.

Reply:
Yep, does make sense for the time being (as it's a private method anyway)